src/brasil/gov/tiles/tiles/groupcarousel.py

```diff
@@ -0,0 +1,75 @@
+# -*- coding: utf-8 -*-
+from brasil.gov.tiles import _
+from collective.cover.interfaces import ITileEditForm
+from collective.cover.tiles.carousel import CarouselTile
+from collective.cover.tiles.list import IListTile
+from collective.cover.widgets.textlinessortable import TextLinesSortableFieldWidget
+from plone.autoform import directives as form
+from Products.Five.browser.pagetemplatefile import ViewPageTemplateFile
+from zope import schema
+from zope.interface import implementer
+
+
+class IGroupCarouselTile(IListTile):
+    """Display a carousel of items."""
+
+    tile_description = schema.Text(title=_(u'Tile Description'), required=False)
+    form.omitted('tile_description')
+    form.no_omit(ITileEditForm, 'tile_description')
+
+    switch_text = schema.TextLine(title=_(u'Switch Text'), required=False)
+    form.omitted('switch_text')
+    form.no_omit(ITileEditForm, 'switch_text')
```

Comment:
We need to think on a better name than *switch_text*;
This text is used to be inserted into tabs when there are more than one tile of this type in the same row/column.
It is a use case that should be documented in this package.

Reply:
@claytonc I can take care of this field changes.

Reply:
Ok

Reply:
@rodfersou the [videocarousel tile also has this field](https://github.com/plonegovbr/brasil.gov.tiles/blob/master/src/brasil/gov/tiles/tiles/videocarousel.py#L20); I prefer to fix this on both tiles a different pull request after this one is merged.